pegasus/sites.v3/code.org/views/video_with_fallback.haml

```diff
@@ -0,0 +1,5 @@
+%div{style: "width:100%; height: 100%; margin: 0 auto;"}
```

Comment:
Nit, not blocking: `%figure` can be used here instead of `%div` to make this more accessibility friendly.

Edit: I accidentally hit the "Start review" button instead of just leaving a comment — apologies!

Reply:
Done! no worries -- "start review" just lets you batch a bunch of comments together in one email update. if you press it by accident, just take one last look at the PR when you are done reviewing to make sure none of the comments you have written are still pending.